net/libreswan/files/etc/firewall.libreswan-rules

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
```

Comment:
OpenWrt now uses firewall4 which is based on nftables.
It would therefore be good if we support both the old firewall3 (iptables) and the new firewall4 (nftables).
Also, "only" IPv4 is set. Don't we also need this for IPv6?

This should also be stored as a separate hotplug script and not be called via ipsec.user.

Reply:
Done